README.md

```diff
@@ -305,16 +493,26 @@ app.post("/users", function(context) {
 }).use([ validateNewUserMiddleware ]);
 ```
 
-See the [Middleware](#middleware) section for more information.
-
 <div id="route_authorize"><h3>authorize(<i>callback</i>)</h3></div>
 
-The *authorize()* method allows you to specify a route-level function to validate or authorize incoming requests. A route-level authorize handler will override the global authorize handler set by [*Rocky.authorize()*](#rocky_authorize) for requests made to the specified route. The callback function takes a [Rocky.Context](#context) object as a parameter, and must return either `true` (if the request is authorized) or `false` (if the request is not authorized).
+This method allows you to specify a route-level function to validate or authorize incoming requests. Such a route-level authorization handler will override the global authorization handler set by Rocky’s [*authorize()*](#rocky_authorize) method for requests made to the specified route.
+
+#### Parameters ####
+
+| Parameter | Type | Required? | Description |
+| --- | --- | --- | --- |
+| *callback* | Function | Yes | A function to authorize or reject the request |
 
-The *authorize()* method is executed before the main request handler.
+The callback function receives a [Rocky.Context](#context) object as its single argument and must return either `true` (if the request is authorized) or `false` (if the request is not authorized). The callback is executed before the main request handler, so:
 
 - If the callback returns `true`, the route handler will be invoked.
-- If the callback returns `false`, the [onUnauthorized](#rocky_onAuthorized) handler is invoked.
+- If the callback returns `false`, the route-specific [onUnauthorized](#route_onAuthorized) response handler is invoked. If there is no route-specific [onUnauthorized](#route_onAuthorized) response handler, the global [onUnauthorized](#rocky_onAuthorized) response handler is invoked.
```

Comment:
all 3 onUnauthorized links don't appear to do anything

Reply:
Fixed multiple instances